### PR TITLE
Fix error with PHP 5.4 and higher

### PR DESCRIPTION
**Pull request by QuickCRM**
render('') generates an error with imagepng function for PHP 5.4 and higher.
Must be replaced by render(NULL)
See http://php.net/manual/fr/function.imagepng.php

## Description

There is no issue in SuiteCRM itself as this function is not used natively
However, QuickCRM uses it to generate charts.
It could also be used by customizations to export the graphs.

## Motivation and Context
This fix allows returning graphs as images

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by pgorod** [comment]
@QuickCRM you should change the branch of this PR to target the `hotfix` branch, instead of `master`. Thanks.

**Comment by QuickCRM** [comment]
Sorry about that. 
We will do it. 

--
support@quickcrm.fr
NS-Team
www.quickcrm.fr/mobile

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-## SuiteCRM 7.9.1
+## SuiteCRM 7.9.2
```
Comment (daniel-samson) [inline README.md]: Please only include the fix in your pull request.

File: modules/AOR_Charts/AOR_Chart.php [hunk 2]
```diff
@@ -222,7 +222,7 @@ public function buildChartImage(array $reportData, array $fields,$asDataURI = tr
             $chartPicture->replaceImageMapTitle("data", $labels);
         }
         ob_start();
-        $chartPicture->render('');
+        $chartPicture->render(NULL);
```
Comment (daniel-samson) [inline modules/AOR_Charts/AOR_Chart.php]: please use a lowercase null.

File: suitecrm_version.php [hunk 3]
```diff
@@ -3,5 +3,5 @@
     die('Not A Valid Entry Point');
 }
 
-$suitecrm_version      = '7.9.1';
-$suitecrm_timestamp    = '2017-06-15 17:00';
\ No newline at end of file
+$suitecrm_version      = '7.9.2';
```
Comment (daniel-samson) [inline suitecrm_version.php]: Again please only include the fix.

**Comment by pgorod** [comment]
@QuickCRM just a tip for you:

If you get into trouble with `git` trying to remove that extra commit, the easiest way in this case, since it's such a simple change, is to edit the same `readme.md` file to put the version back to `7.9.1` and commit. This way the two commits will annul each other and the PR won't show any change for that file.

**Comment by QuickCRM** [comment]
I certainly can do that but the PHP documentation spells it NULL.
http://php.net/manual/fr/language.types.null.php <http://php.net/manual/fr/language.types.null.php>


—
support@quickcrm.fr
www.quickcrm.fr/mobile <http://www.quickcrm.fr/mobile>
Visit our FAQ <http://www.quickcrm.fr/mobile/en/content/16-faq>

**Comment by QuickCRM** [comment]
Thanks,

Sorry for the trouble.
All these issues come from the fact that I edited your master branch instead of the hotfix branch.

—
support@quickcrm.fr
www.quickcrm.fr/mobile <http://www.quickcrm.fr/mobile>
Visit our FAQ <http://www.quickcrm.fr/mobile/en/content/16-faq>

**Pull request merged**